packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1791,6 +1791,8 @@ describe("normalizeAndValidateConfig()", () => {
 									max_batch_timeout: null,
 									max_retries: "hello",
 									dead_letter_queue: 5,
+									concurrency_enabled: true,
```

Comment:
@OilyLime - since we want to make `concurrency_enabled` the default going forward, should this be the inverse?

Reply:
Will do.